config/initializers/sidekiq.rb

```diff
@@ -5,6 +5,7 @@
 redis_url = ENV.fetch('REDIS_URL', default_redis_url)
 
 Sidekiq.configure_server do |config|
+  puts "Configuring the sidekiq server to use #{redis_url}"
```

Comment:
Do we need this?

Reply:
just spotted that myself - no Ill remove it - I added it when problem solving